blocks/library/paragraph/index.js

```diff
@@ -158,8 +161,10 @@ registerBlockType( 'core/paragraph', {
 					} }
 					value={ content }
 					onChange={ ( nextContent ) => {
+
 						setAttributes( {
-							content: nextContent,
+							content: nextContent.content,
+							footnotes: nextContent.footnotes,
```

Comment:
This is causing paragraph blocks to include an empty `footnotes` object in comment serialized block attributes whenever the text is changed (regardless of adding footnotes). We should avoid this being included for paragraphs which aren't using footnotes.